### DISCUSSION TITLE
Bibliography

### DISCUSSION THREAD

**Discussion by JordiBolibar** [ODINN-SciML/ODINN.jl#34] [General]
List of papers and content relevant to **ODINN**. You can comment here to make new suggestions and I will add them to this first post to make the list grow. 

***
- [Universal Differential Equations for Scientific Machine Learning](https://arxiv.org/abs/2001.04385). Rackauckas et al. (2020). _arXiv_.

> This is the baseline paper for all this work. This describes the method to combine mechanistic models in the form of differential equations with neural networks, in order to learn subparts an equation. This opens the door to learning missing information in physical models, and to discover new laws and parametrizations. 

***

- [Stiff Neural Ordinary Differential Equations](https://arxiv.org/abs/2103.15341). Kim et al. (2021). _Chaos_.

> This paper analyses the difficulties in training neural differential equations (NDEs) with stiff differential equations. The basic approach from the Neural ODEs paper using a Backsolve adjoint doesn't work except for the simplest cases. In order to correctly obtain gradients for more complex systems, Interpolating or Quadrature adjoints are presented, which are more stable and robust. Other strategies are discussed on how to successfully train these harder NDEs. 

***

- [A Comparison of Automatic Differentiation and Continuous Sensitivity Analysis for Derivatives of Differential Equation Solutions](https://arxiv.org/abs/1812.01892). Ma et al. (2021) ._arXiv_. 

> Benchmark of the different backpropagation techniques for Universal Differential Equations. Goes in the same direction as the "Stiff Neural Ordinary Differential Equations" paper above. 

***

- [Discovering governing equations from data by sparse identification of nonlinear dynamical systems](http://www.pnas.org/cgi/doi/10.1073/pnas.1517384113). Brunton et al. (2016). _Proceedings of the National Academy of Sciences_.

> Paper describing SINDy, the method used to derive a mathematical expression from nonlinear systems (in our case, neural networks). This will be used as the last step in order to transform the learnt nonlinear functions by neural networks into mathematical expressions which can be used to rewrite the original equation of the physical process.  
***

- [Nonlinear sensitivity of glacier mass balance to future climate change unveiled by deep learning](https://www.nature.com/articles/s41467-022-28033-0). Bolibar et al. (2022). _Nature Communications_.

> This paper justifies why we need to work towards a nonlinear parametrization for empirical glacier mass balance models. 
***

- [The Open Global Glacier Model (OGGM)](https://gmd.copernicus.org/articles/12/909/2019/). Maussion et al. (2019). _Geophysical Model Development_. 

> Paper describing OGGM. For more details regarding the model, please check the [docs](https://docs.oggm.org/en/stable/index.html). 
***
